lib/controller.php

```diff
@@ -185,6 +185,14 @@ public function import_blob($blob) {
 			'_sha'         => $blob->sha,
 		) );
 
+		if(class_exists("WPCom_Markdown")) {
+			$wpcomMd = WPCom_Markdown::get_instance();
+			$args = $wpcomMd -> wp_insert_post_data($args, array(
+				"ID"           => $this->id,
+				"post_type"    => $args['post_type']
```

Comment:
Yes, actually, my mistake; [layout](https://github.com/benbalter/wordpress-github-sync/blob/master/lib/post.php#L250) is the `post_type`, so use that.